docs/source/roadmaps/may-2024.md

```diff
@@ -0,0 +1,117 @@
+# May 2024
+This roadmap is split into "themes" which roughly mirror our current priorities.
+
+## Data management
+**Current status**
+
+We have developed [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/), a simple, standard data folder 
+specification for (systems) neuroscience and [datashuttle](datashuttle.neuroinformatics.dev), to automate the 
+creation, validation and transfer of NeuroBlueprint projects. For more details on this specific project, see the 
+[datashuttle roadmap]().
+
+**Q4 2024**
+
+Standardised metadata support
+
+**Q2 2025**
+
+Support for customising data structure (e.g. specific mandatory elements or new file types)
```

Comment:
Is this like the ability to have customised versions of NeuroBlueprint and a mechanism by which to switch between them? (e.g. BIDS 2.0)?

Reply:
yeah, or even have a fully custom version (i.e. doesn't really resemble neuroblueprint at all) if possible

Reply:
I think at this stage I'd rather it not be on the priority list, as it would be quite hard to implement. In the short term it's definitely useful to provide converters from a lab structure to NB, or between major specifications (e.g. BIDS, ONE), and have some basic custom validation (e.g. name templates). But I think allowing users to create arbitrary structures and convert to NB on the fly is risky (if I have understood correctly this is the point).

Reply:
if you think it will take a long time, that's fine, lets just push back the date

Reply:
That's interesting it would be good to discuss the specifics further, I'll open an issue on the NeuroBlueprint repo. In that case how about we swap NWB and the customisation, that is at least how I would prioritise these (I think NWB conversion should be much easier).

Reply:
I've raised an issue, I will tidy up the datashuttle / NB roadmap and tag you. Once that's finalised we can just copy the deliverables from there to here?

Reply:
They probs shouldn't be copied (vs summarised at a high level), unless there are very few?

Reply:
That makes sense, please see revisions to roadmap [described here](https://github.com/neuroinformatics-unit/datashuttle/pull/385#discussion_r1603888136), they are not so many but happy for them to be summarised in any way that fits with the NIU roadmap.

Reply:
Maybe if you agree with splitting NB and datashuttle roadmaps it can be brief summary of both combined.